r/R/csv.R

```diff
@@ -722,9 +731,10 @@ write_csv_arrow <- function(x,
 
   if (is.null(write_options)) {
     write_options <- readr_to_csv_write_options(
-      include_header = include_header,
-      batch_size = batch_size
+      col_names = include_header,
+      na = na
     )
+    # TODO: what to do with batch_size??
```

Comment:
I think I mostly have it, but the model I'm going for requires the `readr_to_csv_*_options` parameter set to be disjoint from the `Csv*Options$create` parameter set. The fact that we have `batch_size` here (when it's already exposed via `write_options`) throws a wrench in that plan 😞 

If there were a `readr` equivalent I would map it do that, but I don't believe there is.

Reply:
I guess part of the issue is that for the `Csv*Options` objects, we have `create()` methods which set up all the options initially, but no methods for updating the different options later.  

What do you think about maybe adding methods for updating these objects?

Reply:
I could do that. This PR might be a little big though :)